services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/validation/HankePublicValidator.kt

```diff
@@ -71,7 +71,6 @@ object HankePublicValidator {
      * - Email
      */
     private fun validateYhteystieto(yhteystieto: HankeYhteystieto, path: String): ValidationResult =
-        validate { notBlank(yhteystieto.etunimi, "$path.etunimi") }
-            .and { notBlank(yhteystieto.sukunimi, "$path.sukunimi") }
+        validate { notBlank(yhteystieto.nimi, "$path.etunimi") }
```

Comment:
Phone number is also a mandatory field now?

Reply:
Hmm phone number actually seems not to be mandatory. According to this at least https://www.figma.com/file/dRtEnlMJUwz0KUPqAaTzq3/Haitaton-2.0?node-id=2287-75423

I'll change this in hankeyhteystieto.

Reply:
Done.